ReleaseHistory.md

```diff
@@ -1,5 +1,6 @@
 # SARIF Package Release History (SDK, Driver, Converters, and Multitool)
 ## **v4.1.0** UNRELEASED
+* BUG: If HDF `code_desc` is empty, use `desc` as the SARIF `message`. [#2632](https://github.com/microsoft/sarif-sdk/pull/2632)
```

Comment:
>BUG [](http://example.com/codeflow?start=2&length=3)

All BUG entries belong together in the list below, since I asked you to review another possible change could you make this one?

Also let's reference `HDFConverter` to make things a little more explicit:

BUG: If `HDFConverter` `code_desc`... etc.

Reply:
I've made that update - how's it look now?

Reply:
Looks great! I'm going to ship this right now for you via NuGet.